examples/plotting/histogram.py

```diff
@@ -6,6 +6,9 @@
     :keywords: histogram
 
 '''
+#Updated all basic colors to their hexadecimal counterparts
+#Updated Warm/Low Contrast Pallette to High Contrast Pallettes
+
```

Comment:
These comments should be removed, because they don't address the right audience. The audience for examples is *users* of of Bokeh, and examples are intended to be kept a as short and laser-focused on a usage topic as possible for their sake. Any of this dev history, etc is not relevant of useful to *users* of Bokeh (and the history is in the commits and GH if *developers* of Bokeh really need to see it)